tests/components/test_wake_on_lan.py

```diff
@@ -0,0 +1,53 @@
+"""Tests for Wake On LAN component."""
+import asyncio
+from functools import partial
+from unittest import mock
+
+import pytest
+
+from homeassistant.setup import async_setup_component
+from homeassistant.components.wake_on_lan import (
+    DOMAIN, SERVICE_SEND_MAGIC_PACKET)
+
+
+@pytest.fixture
+def mock_wakeonlan():
+    """Mock mock_wakeonlan."""
+    module = mock.MagicMock()
+    with mock.patch.dict('sys.modules', {
+        'wakeonlan': module,
+    }):
+        yield module
+
+
+@asyncio.coroutine
+def test_setup_component(hass, mock_wakeonlan):
```

Comment:
This test is kinda useless as the service existence is also tested in the other test.